lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystemE2E.cs

```diff
@@ -269,6 +272,21 @@ public void TestCopyFromLocalE2E()
             container.DeleteIfExistsAsync().Wait();
         }
 
+        [Fact(Skip = SkipMessage)]
+        public void TestIsDirectory()
```

Comment:
We test for several different conditions here. Maybe split this test into several independent tests?

Reply:
fixed

---
In reply to: [198682001](https://github.com/apache/reef/pull/1473#discussion_r198682001) [](ancestors = 198682001)